Unhandled exceptions

Hi!

I added this GitHub Action, but I get an error when I push:
```
Unhandled exception. System.InvalidOperationException: fatal: no such path mediaportal/Build Release.bat	 in f6078a69b2700e8a41bcdadddc12a72be1239d90

   at devrating.git.GitProcess.Output()
   at devrating.git.GitAFileBlames.InitedBlames()
   at devrating.git.GitAFileBlames.AtLine(UInt32 line)
   at devrating.git.GitDeletions.HunkDeletions(String header)+MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at devrating.git.GitDeletions.Items()
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Text.Json.Serialization.Converters.IEnumerableDefaultConverter`2.OnWriteResume(Utf8JsonWriter writer, TCollection value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.Serialization.JsonCollectionConverter`2.OnTryWrite(Utf8JsonWriter writer, TCollection value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.Serialization.JsonConverter`1.TryWrite(Utf8JsonWriter writer, T& value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.Serialization.Metadata.JsonPropertyInfo`1.GetMemberAndWriteJson(Object obj, WriteStack& state, Utf8JsonWriter writer)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryWrite(Utf8JsonWriter writer, T value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.Serialization.JsonConverter`1.TryWrite(Utf8JsonWriter writer, T& value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.Serialization.JsonConverter`1.WriteCore(Utf8JsonWriter writer, T& value, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.JsonSerializer.WriteCore[TValue](Utf8JsonWriter writer, TValue& value, JsonTypeInfo`1 jsonTypeInfo)
   at System.Text.Json.JsonSerializer.WriteString[TValue](TValue& value, JsonTypeInfo`1 jsonTypeInfo)
   at System.Text.Json.JsonSerializer.Serialize[TValue](TValue value, JsonSerializerOptions options)
   at devrating.git.GitDiff.ToJson()
   at StabilityMetric.Apply(Diff diff) in /home/runner/work/_actions/victorx64/pr-label/v0/core/StabilityMetric.cs:line 48
   at Program.<Main>$(String[] args) in /home/runner/work/_actions/victorx64/pr-label/v0/on-push/Program.cs:line 28
   at Program.<Main>(String[] args)
Error: Process completed with exit code 1.
```

And at Pull Request I get this error:
```
Unhandled exception. Octokit.ForbiddenException: Resource not accessible by integration
   at Octokit.Connection.HandleErrors(IResponse response) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 704
   at Octokit.Connection.RunRequest(IRequest request, CancellationToken cancellationToken) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 685
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 670
   at Octokit.ApiConnection.Post[T](Uri uri, Object data, String accepts, String contentType, CancellationToken cancellationToken) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/ApiConnection.cs:line 293
   at SizeLabel.CreateLabelsForRepository() in /home/runner/work/_actions/victorx64/pr-label/v0/core/SizeLabel.cs:line 64
   at SizeLabel.Update(UInt32 additions) in /home/runner/work/_actions/victorx64/pr-label/v0/core/SizeLabel.cs:line 30
   at GitHubClient.UpdatePrLabels(UInt32 size, Double rating, Int32 prNumber) in /home/runner/work/_actions/victorx64/pr-label/v0/core/GitHubClient.cs:line 78
   at Program.<Main>$(String[] args) in /home/runner/work/_actions/victorx64/pr-label/v0/on-pull_request/Program.cs:line 65
   at Program.<Main>(String[] args)
{"message":"Resource not accessible by integration","documentation_url":"https://docs.github.com/rest/reference/issues#create-a-label"}
Error: Process completed with exit code 1.
```

Is there any way to solve this?

--- a/README.md
+++ b/README.md
@@ -34,6 +34,9 @@ name: Add PR size and stability labels
     branches: [master, main]
 jobs:
   add-pr-labels:
+    permissions:
+      contents: read
+      pull-requests: write
     runs-on: ubuntu-latest
     steps:
       - uses: victorx64/pr-label@v0

--- a/core/core.csproj
+++ b/core/core.csproj
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="devrating.factory" Version="23.1.1" />
-    <PackageReference Include="devrating.git" Version="23.3.1" />
-    <PackageReference Include="devrating.sqlite" Version="23.1.1" />
+    <PackageReference Include="devrating.factory" Version="23.3.1" />
+    <PackageReference Include="devrating.git" Version="23.4.0" />
+    <PackageReference Include="devrating.sqlite" Version="23.3.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
     <PackageReference Include="Octokit" Version="4.0.3" />
     <PackageReference Include="Octokit.GraphQL" Version="0.2.0-beta" />

